.circleci/config.yml

```
@@ -33,6 +33,38 @@ jobs:
         environment:
           PYTEST_FILTER: notebooks
 
+  format-checker:
+    docker:
+      - image: awav/tensorflow:2.0-preview
```

just noticed we are still using the old docker images?
```suggestion
      - image: awav/tensorflow:2.1
```

---

@vdutor, I cannot apply it here, could you create a PR to develop? Btw, the docker will pull It new tensorflow, it doesn't have pre-installed tensorflow.